spec/unit/services/add_item_to_basket_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "../../../lib/services/add_item_to_basket"
+
+module Shop
+  BASKET = []
+end
+
+RSpec.describe Shop::AddItemToBasket do
+  subject(:basket) { Shop::AddItemToBasket.new }
+
+  before do
+    Shop::BASKET.clear
```

Comment:
hmm I guess not. Thanks for pointing that out. Do they have the same value and it doesn't matter which one I delete?